dockerfile

```diff
@@ -8,4 +8,5 @@ COPY . .
 
```

Comment:
Can we rename this to `Dockerfile` capital D? Without this the casing can cause problems especially on Windows.